Library/Formula/qt-gstreamer.rb

```diff
@@ -0,0 +1,66 @@
+class QtGstreamer < Formula
+  desc "C++ bindings for GStreamer with a Qt-style API"
+  homepage "http://gstreamer.freedesktop.org/modules/qt-gstreamer.html"
+  url "http://gstreamer.freedesktop.org/src/qt-gstreamer/qt-gstreamer-1.2.0.tar.xz"
+  sha256 "9f3b492b74cad9be918e4c4db96df48dab9c012f2ae5667f438b64a4d92e8fd4"
+  head "git://anongit.freedesktop.org/gstreamer/qt-gstreamer"
+
+  option "with-static", "Build QtGStreamer as a static library"
+  option "with-tests", "Build QtGStreamer's tests"
+  option "without-examples", "Build QtGstreamer's examples"
```

Comment:
I did. By default QtGStreamers default build options are chosen. Only when you pass --with-/--without-options, you override the default.